File: Modules/ServerTypes/EmployeeProfile/MasterCareerPathType.ts
import { Decorators } from "@serenity-is/corelib";

export enum MasterCareerPathType {
    increment = 0,
    decrement = 1,
    promotion = 2,
    demotion = 3
}
Decorators.registerEnumType(MasterCareerPathType, 'HRMSoftware.EmployeeProfile.MasterCareerPathType', 'HumanResource.EmployeeProfile.MasterCareerPathType');

File: Modules/HumanResource/EmployeeProfile/MasterCareerPath/MasterCareerPathType.cs
using System.ComponentModel;

namespace HRMSoftware.EmployeeProfile;
[Serenity.ComponentModel.EnumKey("HumanResource.EmployeeProfile.MasterCareerPathType")]
public enum MasterCareerPathType
{
    [Description("INCREMENT")]
    increment = 0,
    [Description("DECREMENT")]
    decrement = 1,
    [Description("PROMOTION")]
    promotion = 2,
    [Description("DEMOTION")]
    demotion = 3,
    [Description("TRANSFER")]
    transfer = 4
}